ESP8266Interface.h

```diff
@@ -161,6 +161,12 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface
      */
     using NetworkInterface::add_dns_server;
 
+    /** Check if connected to the network
+     *
+     *  @return         true if connected, false otherwise
+     */
+    bool isConnected(void);
```

Comment:
This can be removed, it was just a convenient function .. There is nothing wrong with having something as fundamental as checking for a connection .. However, I am not so keen on it .. Can be removed